### PR TITLE
Update where available section of PS overview

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/303843333

Changes proposed in this pull request:

- Update where available section of PS overview
- https://pr-7525-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/overview/
-

### CODE REVIEW

**Comment by Pejmanam** [comment]
`7525`

File: site/en/docs/privacy-sandbox/overview/index.md [hunk 1]
```diff
@@ -133,18 +133,42 @@ the [Privacy Community Group](https://www.w3.org/community/privacycg/participant
...
 
-You can
-[download Chromium](http://chromium.org/getting-involved/download-chromium),
-then [run it with flags](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)
-to allow access to APIs that are in the process of implementation.
+[Chrome Platform Status](chromestatus.com) details browser and implementation status on desktop, Android, iOS, and Android WebView, for individual APIs:
+* [All features](https://chromestatus.com/features) provides shipping  milestones, standardization signals, relevant [Blink](https://www.youtube.com/watch?v=CVRLeOX-TGI) 
+  components, and other information for an individual feature. You can star a feature to be notified of  updates,  and view your starred features from [My features](https://chromestatus.com/myfeatures).
+* [HTML & JavaScript usage metrics](https://chromestatus.com/metrics/feature/popularity) 
+shows the percentage of page loads in Chrome that use a feature across all channels and
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/overview/index.md:151]:
```suggestion
show the percentage of page loads in Chrome that use a feature across all channels and 
```

File: site/en/docs/privacy-sandbox/overview/index.md [hunk 2]
```diff
@@ -133,18 +133,42 @@ the [Privacy Community Group](https://www.w3.org/community/privacycg/participant
...
+as well as [CHIPS](/docs/privacy-sandbox/chips/) and 
+[Related Website Sets](/docs/privacy-sandbox/related-website-sets/), 
+are now available for all users for all Chrome channels on Android and desktop. As with all technologies, use feature detection to check for support and availability before accessing a Privacy Sandbox feature.
 
-You can
-[download Chromium](http://chromium.org/getting-involved/download-chromium),
-then [run it with flags](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)
-to allow access to APIs that are in the process of implementation.
+[Chrome Platform Status](chromestatus.com) details browser and implementation status on desktop, Android, iOS, and Android WebView, for individual APIs:
+* [All features](https://chromestatus.com/features) provides shipping  milestones, standardization signals, relevant [Blink](https://www.youtube.com/watch?v=CVRLeOX-TGI)
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/overview/index.md:148]:
```suggestion
* [All features](https://chromestatus.com/features) provide shipping  milestones, standardization signals, relevant [Blink](https://www.youtube.com/watch?v=CVRLeOX-TGI) 
```

File: site/en/docs/privacy-sandbox/overview/index.md [hunk 3]
```diff
@@ -133,18 +133,42 @@ the [Privacy Community Group](https://www.w3.org/community/privacycg/participant
...
-You can
-[download Chromium](http://chromium.org/getting-involved/download-chromium),
-then [run it with flags](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)
-to allow access to APIs that are in the process of implementation.
+[Chrome Platform Status](chromestatus.com) details browser and implementation status on desktop, Android, iOS, and Android WebView, for individual APIs:
+* [All features](https://chromestatus.com/features) provides shipping  milestones, standardization signals, relevant [Blink](https://www.youtube.com/watch?v=CVRLeOX-TGI) 
+  components, and other information for an individual feature. You can star a feature to be notified of  updates,  and view your starred features from [My features](https://chromestatus.com/myfeatures).
+* [HTML & JavaScript usage metrics](https://chromestatus.com/metrics/feature/popularity) 
+shows the percentage of page loads in Chrome that use a feature across all channels and 
+platforms. Note: this reflects how often the API is being called from pages, not how many users
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/overview/index.md:152]:
```suggestion
platforms. Note that this reflects how often the API is being called from pages, not how many users 
```

File: site/en/docs/privacy-sandbox/overview/index.md [hunk 4]
```diff
@@ -133,18 +133,42 @@ the [Privacy Community Group](https://www.w3.org/community/privacycg/participant
...
+
+### MacOS, iOS and iPadOS
+Chrome on MacOS supports the Privacy Sandbox APIs, but Safari on MacOS (which uses WebKit) does not.
+
+[Apple's App Store rules](https://developer.apple.com/app-store/review/guidelines/#:~:text=2.5.6) mean that Chrome and other browsers on iOS and iPadOS must use the WebKit browser engine, which does not support the Privacy Sandbox APIs. 
+
+### Android WebView
+
+[WebView](https://developer.android.com/reference/android/webkit/WebView) allows you to display web content in an app, but lacks some of the features of 
+fully developed browsers. We do not currently include WebView at this stage of third-party
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/overview/index.md:165]:
```suggestion
fully developed browsers. We don't currently include WebView at this stage of third-party 
```

**Review by anusmitaray — APPROVED** [review]

File: site/en/docs/privacy-sandbox/overview/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/overview/index.md:151]: Name of the page, though

**Comment by nancymic2** [comment]
Changed to the All features section of the Chrome status page

**Pull request merged**